Sources/Peasy/Internal/Socket.swift

```diff
@@ -30,9 +30,26 @@ final class Socket {
 		address.sin6_port = UInt16(port).bigEndian
 		address.sin6_addr = .localhost
 		let size = socklen_t(MemoryLayout<sockaddr_in6>.size)
-		let success = withUnsafePointer(to: &address) { $0.withMemoryRebound(to: sockaddr.self, capacity: Int(size)) { Darwin.bind(tag, $0, size) >= 0 } }
+		let success = withUnsafePointer(to: &address) { address -> Bool in
```

Comment:
Can we use type inference here to remove the `-> Bool` and also the `return` from L34? Not a big deal if not, just a bit tidier

Reply:
Done